docs/src/man/getting_started.md

```diff
@@ -2,88 +2,109 @@
 
 ## Installation
 
-The DataTables package is available through the Julia package system. Throughout the rest of this tutorial, we will assume that you have installed the DataTables package and have already typed `using NullableArrays, DataTables` to bring all of the relevant variables into your current namespace. In addition, we will make use of the `RDatasets` package, which provides access to hundreds of classical data sets.
+The DataTables package is available through the Julia package system. Throughout the rest of this tutorial, we will assume that you have installed the DataTables package and have already typed `using DataTables` to bring all of the relevant variables into your current namespace.
 
-## The `Nullable` Type
+## The `Null` Type
 
-To get started, let's examine the `Nullable` type. Objects of this type can either hold a value, or represent a missing value (`null`). For example, this is a `Nullable` holding the integer `1`:
+To get started, let's examine the `Null` type. `Null` is a type implemented by [Nulls.jl](https://github.com/JuliaData/Nulls.jl) to represent missing data. `null` is an instance of the type `Null` used to represent a missing value. Note that Nulls.jl is not loaded by default when using DataTables and must be imported separately.
 
 ```julia
-Nullable(1)
-```
+julia> using Nulls
 
-And this represents a missing value:
-```julia
-Nullable()
-```
+julia> null
+null
 
-`Nullable` objects support all standard operators, which return another `Nullable`. One of the essential properties of `null` values is that they poison other items. To see this, try to add something like `Nullable(1)` to `Nullable()`:
+julia> typeof(null)
+Nulls.Null
 
-```julia
-Nullable(1) + Nullable()
 ```
 
-The `get` function can be used to extract the value from the [`Nullable`](http://docs.julialang.org/en/stable/manual/types/#nullable-types-representing-missing-values) wrapper when it is not null. For example:
+The `Null` type lets users create `Vector`s and `DataTable` columns with missing values. Here we create a vector with a null value and the element-type of the returned vector is `?Int64`. The syntax `?T`, where `T` can be any type, represents `Union{T, Null}`.
```

Comment:
@quinnj has removed uses of `?T` in the CategoricalArrays PR since it's deprecated on 0.7. Better stop using it too for now, until `T?` is available in 1.0.

Though we'll have to remove the `show` methods from Nulls.jl too if we don't want it to appear in the output.